section1/reflection.md

```diff
@@ -2,18 +2,43 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+I thought I waited too long to ask questions but one thing I realized is the difference between productive and unproductive problem-solving. As long as I'm utilizing the tools at my disposal, like Google, lecture/my notes, Slack search, and feel that I'm making meaningful progress in finding an answer, that is productive problem-solving. It crosses into unproductive territory when I continue to hit an error message trying all of the solutions I have found. According to the articles and Turing's guidance, I should spend no more than 20-30 minutes in this unproductive space. Deciding when something becomes unproductive is what I'm still figuring out. I'm getting there though!
```

Comment:
Great reflection! It can be difficult to find that sweet spot but it seems like you're already very self-aware with it, "still figuring it out" is a great place to be!!